src/v/redpanda/admin/migrations.cc

```diff
@@ -104,102 +104,66 @@ void write_migration_as_json(
 json::validator make_migration_validator() {
     const std::string schema = R"(
 {
+    "$schema": "https://json-schema.org/draft/2020-12/schema#",
```

Comment:
This version isn't supported by rapidjson.
```suggestion
    "$schema": "http://json-schema.org/draft-04/schema#",

Reply:
Thanks. I was going to skip it at all but forgot to hit save in the IDE :facepalm:
I'll leave https in place because ~we need to download schema meta-schemas securely~ that's apparently its URL according to the website and it redirects there anyway.